docs/components/authentication/google-one-tap.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: <GoogleOneTap /> component
+description: Clerk's <GoogleOneTap /> component renders a UI for authenticating users with Google's One Tap API.
+---
+
+# `<GoogleOneTap />` component
+
+> To use Google One Tap with Clerk, you must [enable Google as a social connection in the Clerk Dashboard](https://clerk.com/docs/authentication/social-connections/google#configuring-google-social-connection) and make sure to use custom credentials.
```

Comment:
Let's utilize our [new callouts](https://github.com/clerk/clerk-docs/blob/main/CONTRIBUTING.md#callouts) 
I think the Important one would be good here! :~)

```suggestion
> [!IMPORTANT]
> To use Google One Tap with Clerk, you must [enable Google as a social connection in the Clerk Dashboard](https://clerk.com/docs/authentication/social-connections/google#configuring-google-social-connection) and make sure to use custom credentials.
```